packages/graphile-utils/__tests__/ExtendSchemaPlugin-pg.test.js

```diff
@@ -718,3 +718,116 @@ it("allows adding a single table entry to a nested type", async () => {
     pgClient.release();
   }
 });
+
+it("allows to retrieve a single scalar value", async () => {
+  const schema = await createPostGraphileSchema(pgPool, ["graphile_utils"], {
+    disableDefaultMutations: true,
+    appendPlugins: [
+      makeExtendSchemaPlugin(build => {
+        const { pgSql: sql } = build;
+        return {
+          typeDefs: gql`
+            extend type User {
+              myCustomScalar: Int! @pgQuery(
+                fragment: ${embed(sql.fragment`(SELECT 100)`)}
+              )
+              myCustomScalarWithFunction: Int! @pgQuery(
+                fragment: ${embed(queryBuilder => sql.fragment`(SELECT 101)`)}
+              )
+              myCustomScalarWithFunctionAndArgument(test: Int!): Int! @pgQuery(
+                fragment: ${embed(
+                  (queryBuilder, args) =>
+                    sql.fragment`(SELECT ${sql.value(args.test)}::integer)`
+                )}
+              )
+            }
+          `,
+        };
+      }),
+    ],
+  });
+  const printedSchema = printSchema(schema);
+  expect(printedSchema).toMatchSnapshot();
```

Comment:
Once https://github.com/graphile/graphile-engine/pull/533 lands, can you rebase this pull request on top of it and simplify this test? You'll no longer need to call `printSchema()` yourself, you can just do `expect(schema).toMatchSnapshot()`. 😄 

I'm not sure when that PR will land, but I wanted to give you a heads-up.

Reply:
will do, please ping me when it lands